src/prefect/runner/storage.py

```diff
@@ -564,6 +565,78 @@ def __eq__(self, __value) -> bool:
         return False
 
 
+class LocalStorage:
+    """
+    Sets the working directory in the local filesystem.
+
+    Parameters:
+        url: Local file path to set the working directory for the flow
```

Comment:
hmm. It feels like `url` is not the best name for this

perhaps we should make a function like `create_storage_from_path` so that we don't have to contort the meaning of `url` in this `create_storage_from_url` context

Reply:
What if we renamed it to something like `create_storage_from_source`? I'm trying to avoid changing to much if the underlying behavior

Reply:
that makes sense. @desertaxle would
probably have the most useful opinion on which would be favorable but i'm could imagine a create_storage_from_source that either calls your new logic or the existing from_url logic depending on the value of source

Reply:
This is done.